extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -481,7 +481,7 @@ function is_non_empty($var){
 *    extract a slice from an  array
 *    see: http://www.php.net/manual/en/function.array-slice.php
 */
-    function arrayslice( &$parser, $arrayid_new , $arrayid , $offset, $length='') {
+    function arrayslice( $parser, $arrayid_new , $arrayid , $offset, $length='') {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)